app/assets/javascripts/to_do_items.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
@vlasiak, Please follow the example of how could the class for todo item look like:
```coffeescript
class TodoItem
  constructor: () ->
    @intialize()

  intialize: () ->
    @bindAddEven()
    @bindCancelEven()

  bindAddEven: () ->
    $('div.list a').click =>
      @expandOnClick()

  bindCancelEven(): () ->
    $('div.list form button').click =>
      @collapseOnClick()

  # it is possible that you would need this method
  add: () ->
    #...

  create: () ->
    # a set of method to handle adding of a just saved item 

    

$ ->
  todoItem = new TodoItem
  # todoItem.initialize() # In case `initialize` should not be a part of instantiation

```

Attach esc key pressing on input also should be moved here + all Javascript code from [creation partial](https://github.com/vlasiak/rails_tada/blob/3-a-user-can-create-a-new-todo-item/app/views/to_do_items/create.js.erb) .